### ISSUE TITLE
EPIC: Prepare the Wins page and wins card randomizer for Homepage Launch

### ISSUE THREAD

**Issue by priyanka02art** [hackforla/website#7664]
### Overview
We need to track existing issues and make new issues to deliver Wins page complete enough for the Homepage launch

#### Details
We need to track and support the final wrap up items for the homepage launch, so that it can be completed.

### Action Items
- [x] List all the issues related to the label
- [x] Identify what open issues have to be resolved before launch, by classifying the issues into the following categories using MoSCoW method in a comment.
   - Must Have
   - Should Have
   - Could Have
   - Won't Have this time
- [x] Add the following details under each issue
   ```
   - why: [ADD WHY ITS IMPORTANT TO TO THE LAUNCH]
   - status: [ADD STATUS e.g., Icebox, Emergent Requests, New Issue Approval, ERs and epics that are ready to be turned into issues, Prioritized Backlog, In Progress (actively working), Questions / In Review]
      - draft: [true OR false]
      - responsible: [ADD GITHUB HANDLES WITHOUT @]
   - role: [ADD ROLE e.g. data scientist, infrastructure, design, design lead, user research, Research Lead, dev leads, data analyst, BA, hfla leadership, back end/devOps, writing, front end, merge team, technical writer]
   ```
- [x] Once you receive sign-off from Staff PM add to `## Relevance of the issues to the Homepage launch`
- [x] Review comments with `Notes carried forward from epic` if it exists, and make sure all previous items are addressed in the top of the issue.
   - If it specifies new issues to make then add explicit action items for making those issue in a new comment on this issue
      - get sign-off on those new issues to make by Staff PM 
- [x] Identify items that need issues opened by looking at the page and features and seeing what is not already addressed by the existing issues or new issues to make list.
- [x] Make the issues specified
- [x] Link them as sub issues to this issue
- [x] Classify them using MoSCoW and add the details (why, status, draft, responsible, role)
- [ ] Get Sign-off and prioritization of the issues by Staff PM
- [ ] If this page has content that must be checked for accuracy before homepage launch, add the detail to the epic for tracking that (resource 1.05)

### Resources/Instructions
- Parent Epic: https://github.com/hackforla/website/issues/2481

#### Used for creating this issue
- 1.01 Label name: `P-Feature: Wins Page`
   - 1.01.01 Issue Tab: [P-Feature: Wins Page](https://github.com/hackforla/website/labels/P-Feature%3A%20Wins%20Page)
   - 1.01.02 Project Board: [P-Feature: Wins Page](https://github.com/orgs/hackforla/projects/86/views/1?filterQuery=+label%3A%22P-Feature%3A+Wins+Page%22)
- 1.02 URL of page on Hack for LA website: https://www.hackforla.org/wins/
- 1.03 URL of link to code search: [repo:hackforla/website NOT language:SVG NOT path:/^_data\/internal\/credits\// wins](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+NOT+language%3ASVG+NOT+path%3A%2F%5E_data%5C%2Finternal%5C%2Fcredits%5C%2F%2F+wins&type=code)
- 1.04 URL of WIKI page: https://github.com/hackforla/website/wiki/Wins-page
   - 1.04.01 URL of future edits to wiki pages: https://github.com/hackforla/website/wiki/How-to-Contribute-to-the-Wiki (check this page to see if there are any other new edits in the queue.
      - 1.04.01.01 https://github.com/hackforla/website/issues/4134#issuecomment-1741851926
      - 1.04.01.02https://github.com/hackforla/website/issues/6680#issuecomment-2072920611
- 1.05 Issue for adding items to a final checklist: #8417

#### Created by this issue
- 2.01 [EXAMPLE would be a link to an issue related to delivering this page]

---

### Must Have
TBD

### Should Have
TBD

### Could Have
TBD

---
## Relevance of the issues to the Homepage launch
last updated: 2026-04-21
Highest number found: 8493

### Must Have
- [ ] #7664
   - why: It is the epic for this page (this issue)
   - status: New Issue Approval
      - draft: True
      - responsible: kimberlyvaldez07 
   - role: product

- [ ] #8492
   - why: We need to change the WINS form to only offer to use their GitHub image. We will not be able to display people's LinkedIn profile pictures so we need to revise the form to reflect that.
   - status: New Issue Approval
      - draft: True
      - responsible: None 
   - role: Front end

- [ ] #7594
   - why: if wins don't show up in the new project board, they won't be reviewed and added to the site
   - status: new issue approval
      - draft: True
      - responsible: none
   - role: backend/devOps
 
### Should Have
- [ ] #8493
   - why: We need to ask people to submit wins to add to the wins page.
   - status: Ready for Prioritization
      - draft: False
      - responsible: None 
   - role: Product and writing

- [ ] #7707
   - why: Once the site is re launched, if we discover a problem during usability testing, we want to be able to fix things fast and this script is notoriously hard for new people to work on.
   - status: Questions / In Review
      - draft: False
      - responsible: daras-cu 
   - role: back end/devOps

- [ ] #6668
   - why: Requires review to determine action. See https://github.com/hackforla/website/issues/6668#issuecomment-3229775578 (ready for dev lead and added to the [dev/pm agenda 2025-08-25](https://github.com/hackforla/website/issues/7859#issuecomment-3221706808) for adding to 2025-09-08
   - status: Ice Box
      - draft: False
      - responsible: None
   - role: front end

- [x] #5470 
   -  why: We need to modify wins.js so that if the existing GitHub/LinkedIn fields (which contain URLs) are not populated (see #5472)
   - status: In progress (actively working)
      - draft: False
      - responsible: lc1715 
   - role: front end 

- [ ] #4388 
   - why: so that post launch we know how people interacted with this page
   - status: ERs and epics that are ready to be turned into issues
      - draft: False
      - responsible: None
   - role: backend/devOps

- [ ] #3145
   - why: social media strategy for announcing launch
   - status: ice box
      - draft: True
      - responsible: None
   - role: product

- [ ] #3142 
   - why: to create a better use experience that has them share it with other people
   - status: New Issue Approval
      - draft: True
      - responsible: None
   - role: product

- [ ] #3140 
   - why: to get more people to complete their wins submissions (instead of dropping out)
   - status: prioritized backlog 
      - draft: False
      - responsible: None
   - role: writing

### Could Have
<details><summary>List of all could have issues</summary>

- #6672
   - why: Is dependent on a different issue being solved and is likely a false positive.
   - status: Ice box
      - draft: False
      - responsible: None
   - role: front end 

- #2148
   - why: Its technical debt and not crucial to the launch, but if we want to change anything about the wins card, or do dynamic card on the about page (which we removed from launch requirement), we will need this issue done.
   - status: Ice box
      - draft: False
      - responsible: None
   - role: back end/devOps

</details> 

### Won't have this time
<details><summary>List of all won't have issues</summary>

- #4671 
- #3627
- #3529
- #3143

</details> 

#### Template
```
- [ ] #
   - why: 
   - status: 
      - draft: 
      - responsible: 
   - role:  
```
<details><summary>List of all issue</summary>

- #8493
- #8492
- #7707
- #7664
- #7594
- #6672
- #6668
- #5470
- #4671
- #4388
- #3627
- #3529
- #3145
- #3143
- #3142
- #3140
- #2148

</details>

**Comment by priyanka02art** [comment]
- URL: https://hackforla.org/wins
- [x] [Wins issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+wins+page%22)
   - [x] Write an issue to manual pick a new win, and hard code it on the about page.  The randomizer can't get worked on until the homepage branch is merged into the gh-pages branch
      - [x] ~https://github.com/hackforla/website/issues/4282 (has a dependency on 4910 being finished)~
      - [x] ~#4190~
   - [x] Create an issue to change the WINS form to only offer to use their GitHub image.  We will not be able to display people's LinkedIn profile pictures, the [LinkedIn API Terms of Service](https://www.linkedin.com/legal/l/api-terms-of-use) disallows it. So we need to revise the form to reflect that.
   - [x] #6272
   - [x] #3140 
   - [x] #4388

**Comment by priyanka02art** [comment]
# List of all issue
- [x] #2147
- [x] #2385
- [x] #3627 
- [x] #3140
- [x] #3142 
- [x] #3143
- [x] #3145
- [x] #3529
- [x] #4134
- [x] #4388
- [x] #4671
- [x] #5470 
- [x] #5472
- [x] #6911
- [x] #7495
- [x] #7594


## Relevance of the issues to the Homepage launch
### Must Have
- #7594
   - why: if wins don't show up in the new project board, they won't be reviewed and added to the site
   - status: new issue approval
      - draft
      - responsible: RW
   - role: backend/devOps
### Should Have
- #3145
   - why: social media strategy for announcing launch
   - status: icebox
   - role: product
- #4388 
   - why: so that post launch we know how people interacted with this page
   - status: Epics and ERs that are ready to be turned into issues
   - role: backend/devOps
- ask people to submit wins 
### Could Have
- #3140 
   - why: to get more people to complete their wins submissions (instead of dropping out)
   - status: prioritized backlog 
   - role: writer
- #3142 
   - why: to create a better use experience that has them share it with other people
   - status: new issue approval
   - role: product
- #5472
   - why: The URLs that appear on the wins page should be consistent, for professionalism
   - status: new issue approval
      - draft
      - responsible: RW
   - role: backend/devOps
- #5470 
   -  see #5472
   - status: icebox
### Won't have this time
- #4671 
- #3627
- #3143
- #3529
- #2385
- #4134
- #2147
- #6911
- #7495

**Comment by aparnar1920** [comment]
# List of all issues
- [ ] #7937 
- [ ] #7707  
- [ ] #7664 
- [ ] #7594 
- [ ] #7569 
- [ ] #6911 
- [ ] #6672 
- [ ] #6668 
- [ ] #6669 
- [ ] #5472 
- [ ] #5470 
- [ ] #4671 
- [ ] #4388 
- [ ] #3627 
- [ ] #3529 
- [ ] #3145 
- [ ] #3143 
- [ ] #3142 
- [ ] #3140 
- [ ] #2385 
- [ ] #2148 
- [ ] #2147


## Relevance of the issues to the Homepage launch
### Must Have
- [ ] #7937 
   - why: For the Wins overlay we need to update the image used for the GitHub logo to icon-github.svg so that it is WCAG compliant and easier to change out the icon for any new icon that is a svg file.
   - status: New Issue Approval
      - draft: False
      - responsible: None 
   - role: Front End
- [ ] #7664    
   - why: It is the epic for this page (this issue)
   - status: New Issue Approval
      - draft: True
      - responsible: None  
   - role: Product
- [ ] #3627 (Confirm if this is only in Figma or on the website also)  
   - why: Need to standardized the dropdown menu for a cohesive user experience.
   - status: New Issue Approval
      - draft: True 
      - responsible: None  
   - role: Design   
- [ ] #3529    
   - why: We need to audit the Wins Page to know which text elements are missing text components so that we can fix them in the Figma, and ensure consistency with the Design System and Implementation by the website team. 
   - status: Prioritized Backlog
      - draft: False 
      - responsible: None  
   - role: Design

### Should Have
- [ ] #7707  
   - why: We need to create documentation to set up current Wins Data for local testing
   - status: In Progress  
      - draft: False
      - responsible: daras-cu  
   - role: Back End/DevOps 
- [ ] #6911     
   - why: To reduce duplicate images, we need to update the Wins page code to use _includes/svg/icon-github.svg instead of /assets/images/wins-page/icon-github-small.svg.
   - status: Questions/In Review
      - draft: False
      - responsible: izma-mujeeb  
   - role: Front End 
- [ ] #4388     
   - why: To know how many people start a Wins submission but don't complete it 
   - status: Prioritized Backlog
      - draft: False 
      - responsible: Cloid   
   - role: Back End/DevOps   
- [ ] #3143     
   - why: Based on the Wins submission research findings, users would like an opportunity to edit their wins after they submit so they have more control over their submitted info. 
   - status: New Issue Approval
      - draft: True 
      - responsible: None  
   - role: Product
- [ ] #3142     
   - why: Based on user research, users would like to be notified via email once their win is published to the hfla website. They would like a link to their published win. The notification process needs to be setup
   - status: New Issue Approval
      - draft: True 
      - responsible: None
   - role: Product

### Could Have
- [ ] #7594     
   - why: Need it for proper issue tracking. 
   - status: New Issue Approval
      - draft: False
      - responsible: roslynwythe   
   - role: Back End/DevOps  
- [ ] #7569     
   - why: To ensure a cleaner and more efficient use of assets 
   - status: PRs being reviewed
      - draft: False
      - responsible: None (9 reviewers)  
   - role: Front end  
- [ ] #6672  
   - why: We need to resolve the new alert [(34)](https://github.com/hackforla/website/security/code-scanning/34) and either recommend dismissal of the alert or update the code files to resolve the alert. 
   - status: Ice Box 
      - draft: False
      - responsible: None 
   - role: Front End
- [ ] #6668  
   - why: We need to resolve the new alert [(49)](https://github.com/hackforla/website/security/code-scanning/49) and either recommend dismissal of the alert or update the code files to resolve the alert. 
   - status: Ice Box  
      - draft: False
      - responsible: None 
   - role: Front End
- [ ] #4671  
   - why: We need a user-friendly way to collect GitHub and LinkedIn URLs, store only the essential profile descriptors for efficiency and reusability, and ensure the URLs are displayed consistently on the Wins webpage.
   - status: Ice Box
      - draft: False
      - responsible: None 
   - role: Front End
- [ ] #    
   - why: 
   - status:
      - draft: 
      - responsible:  
   - role: 

### Won't have this time
- [ ] #6669 
   - why: We need to resolve the new alert (48) and either recommend dismissal of the alert or update the code files to resolve the alert. The alert addresses a missing variable declaration in google-apps-scripts/wins-form-responses/Code.js
   - status: New Issue Approval  
      - draft: False
      - responsible: roslynwythe  
   - role: Back End/Dev Ops
- [ ] #5472  
   - why: To streamline data handling, we need new fields for GitHub and LinkedIn handles (rather than full URLs) and to integrate them into the Response sheet, Review sheet, and the _wins-data.json feed.
   - status: In Progress  
      - draft: False
      - responsible: siyunfeng  
   - role: Back End/DevOps
- [ ] #5470     
   - why: We need to modify wins.js so that if the existing GitHub/LinkedIn fields (which contain URLs) are not populated, the code will use the new fields GitHub-handle and LinkedIn-name and reconstruct the appropriate URLs.
   - status: Ice Box
      - draft: False
      - responsible: None 
   - role: Front End   
- [ ] #3145     
   - why: Please reference UX research results re: Win social media strategy when HfLA decides to pick up social media 
   - status: Ice Box
      - draft: False
      - responsible: None 
   - role: Product 

#### Templates
```
- [ ] # 
   - why: 
   - status: 
      - draft
      - responsible: 
   - role: 
```
```
### Issues list associated with this epic
last updated: 
Highest number found:
```

**Comment by nadiaspn0503** [comment]
### Issues list associated with this epic
last updated: 2025-08-24
Highest number found: 8289
- #8289
- #8147
- #7937
- #7707 
- #7664
- #6672
- #6668
- #2148

--- 
### Must have
 - #7664
   - why: track existing issues and make new issues to deliver Wins page complete enough for the Homepage launch
   - status: New Issue Approval
      - draft: True
      - responsible: None
   - role: product
   
- #8289
   - why: A crashing page disrupts the user experience.
   - status: Prioritized Backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #8147
   - why: A crashing page disrupts the user experience.
   - status: Closed
      - draft: False
      - responsible: @DDVVPP 
   - role: back end/devOps and front end

- #7937
   - why: Image must be WCAG compliant
   - status: New Issue Approval (ready for dev lead and added to the [dev/pm agenda 2025-08-25](https://github.com/hackforla/website/issues/7859#issuecomment-3221706808) for adding to 2025-09-08
      - draft: True
      - responsible: None
   - role: front end
  
### Should have
- #7707
   - why: Once the site is re launched, if we discover a problem during usability testing, we want to be able to fix things fast and this script is notoriously hard for new people to work on.
   - status: In progress (actively working) - waiting on a review of documentation from Bonnie
      - draft: False
      - responsible: @daras-cu 
   - role: back end/devOps

- #6668
   - why: Requires review to determine action. See https://github.com/hackforla/website/issues/6668#issuecomment-3229775578 (ready for dev lead and added to the [dev/pm agenda 2025-08-25](https://github.com/hackforla/website/issues/7859#issuecomment-3221706808) for adding to 2025-09-08
   - status: Ice Box
      - draft: False
      - responsible: None
   - role: front end

### Could have
- #6672
   - why: Is dependent on a different issue being solved and is likely a false positive.
   - status: Ice box
      - draft: False
      - responsible: None
   - role: front end

- #2148
   - why: Its technical debt and not crucial to the launch, but if we want to change anything about the wins card, or do dynamic card on the about page (which we removed from launch requirement), we will need this issue done.
   - status: Ice box
      - draft: False
      - responsible: None
   - role: back end/devOps

### Won't have

**Comment by ExperimentsInHonesty** [comment]
I think there is a conflict in these issues. I want to sort it out before I prioritize the open issues.

This issue is part of a whole restructuring of GitHub logo icon
- https://github.com/hackforla/website/issues/7937. (a sub issue of 7727)

### Other related issues
- #6903 (this issue moves the icons into the _includes/svg/ folder and removes duplicate icons
   - #6908
   - #6909
   - #6910
   - #6911
- https://github.com/hackforla/website/issues/7727
   - https://github.com/hackforla/website/issues/7937
- https://github.com/hackforla/website/issues/6911
- ~#7919 (duplicate of 7937)~
- #2567
   - #3879
   - #2929
   - #3889

#### Probably not relevant
- #6161

- Code files icon-github.svg appears in: [GitHub Search icon-github.svg&type=code](https://github.com/search?q=repo%3Ahackforla%2Fwebsite%20icon-github.svg&type=code)
- Issue files icon-github.svg appears in: [GitHub Search icon-github.svg&type=issues](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+icon-github.svg&type=issues&p=1)

**Comment by ExperimentsInHonesty** [comment]
Once we have identified all the Open issues related to this Epic, we will go to the wins page and visually identify any other issues that might need to be made.  https://hackforla.org/wins

**Comment by nadiaspn0503** [comment]
### Issues list associated with this epic
last updated: 2025-08-29.
Highest number found: 8280
- #8280
- #8279
- #8145

--- 
### Must have
 - #8280
   - why: New wins form submission must be reviewed
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: Product

 - #8279
   - why: New wins form submission must be reviewed
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: Product

 - #8145
   - why: New wins form submission must be reviewed
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: Product

### Should have

### Could have

### Won't have

**Comment by kimberlyvaldez07** [comment]
2026-08-18

Using this search 
[state:open label:"P-Feature: Wins Page" is:issue ](https://github.com/hackforla/website/issues?q=state%3Aopen%20label%3A%22P-Feature%3A%20Wins%20Page%22%20is%3Aissue) 

This is the highest issue number found 8701 it is addressed at the top of this issue.
